__tests__/components/Comments/Comment.test.js

```diff
@@ -96,7 +97,8 @@ describe('Comment test should run correctly', () => {
         findCommentAndUpdate={findComment}
         updateVote={updateVote}
         time={time}
-      />
+      />,
+      {wrapper: Store}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Store` context is now being passed as a wrapper prop in the test render function. This change is likely made to provide necessary context values to the `Comment` component during testing. Make sure that the `Store` context is properly initialized with all necessary default values for the tests to run correctly.
```diff
+       />,
+       {wrapper: Store}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->